frontend/public/locales/ja/cluster-settings.json

```diff
@@ -1,48 +1,50 @@
 {
-  "Invalid cluster version": "無効なクラスターバージョン",
-  "Cancel update": "更新をキャンセル",
-  "Available updates": "利用可能なアップデート",
-  "Failing": "失敗する",
-  "View conditions": "条件を見る",
-  "Update to {{version}} in progress": "に更新{{version}}進行中",
+  "Invalid cluster version": "無効なクラスターのバージョン",
+  "Cancel update": "更新のキャンセル",
+  "Available updates": "利用可能な更新",
+  "Failing": "失敗",
+  "View conditions": "状態の表示",
+  "Update to {{version}} in progress": "{{version}} への更新が進行中",
   "No update channel selected": "更新チャネルが選択されていません",
   "Version not found": "バージョンが見つかりません",
   "Error retrieving": "取得エラー",
-  "Up to date": "最新の",
-  "View release notes": "リリースノートを見る",
+  "Up to date": "最新",
+  "View release notes": "リリースノートの表示",
   "Unknown": "Unknown",
   "None": "なし",
-  "Current version": "現行版",
-  "Last completed version": "最後に完成したバージョン",
-  "Learn more about OpenShift update channels": "OpenShiftアップデートチャネルの詳細",
+  "Current version": "現在のクラスター",
+  "Last completed version": "最後に完了したバージョン",
+  "Learn more about OpenShift update channels": "OpenShift 更新チャネルの詳細",
   "Channel": "チャネル",
-  "Channels help to control the pace of updates and recommend the appropriate release versions. Update channels are tied to a minor version of OpenShift Container Platform, for example 4.5.": "チャネルは、更新のペースを制御し、適切なリリースバージョンを推奨するのに役立ちます。更新チャネルは、OpenShift ContainerPlatformのマイナーバージョン（4.5など）に関連付けられています。",
+  "Channels help to control the pace of updates and recommend the appropriate release versions. Update channels are tied to a minor version of OpenShift Container Platform, for example 4.5.": "チャネルは、更新のペースを制御し、適切なリリースバージョンを推奨するのに役立ちます。更新チャネルは、OpenShift ContainerPlatform のマイナーバージョン (4.5 など) に関連付けられています。",
   "Version": "バージョン",
-  "Worker nodes": "ワーカーノード",
-  "Worker nodes may continue to update after the update of master nodes and operators are complete.": "ワーカーノードは、マスターノードとオペレーターの更新が完了した後も更新を継続できます。",
-  "{{updatedWorkerNodes}} of {{totalWorkerNodes}}": "{{updatedWorkerNodes}} の {{totalWorkerNodes}}",
-  "+ More": "+その他",
+  "{{name}} {{resource}} may continue to update after the update of Master {{resource}} and {{resource2}} are complete.": "{{name}} {{resource}} は、マスター {{resource}} と {{resource2}} の更新が完了した後も更新を継続できます。",
+  "{{updatedMCPNodes}} of {{totalMCPNodes}}": "{{updatedMCPNodes}}/{{totalMCPNodes}}",
+  "+ More": "+ その他",
   "{{currentChannel}} channel": "{{currentChannel}} チャネル",
   "{{newerChannel}} channel": "{{newerChannel}} チャネル",
-  "Cluster operators": "クラスターオペレーター",
-  "{{updatedOperatorsCount}} of {{totalOperatorsCount}}": "{{updatedOperatorsCount}} の {{totalOperatorsCount}}",
-  "Master nodes": "マスターノード",
-  "{{updatedMasterNodes}} of {{totalMasterNodes}}": "{{updatedMasterNodes}} の {{totalMasterNodes}}",
-  "Update status": "最新状況",
+  "{{updatedOperatorsCount}} of {{totalOperatorsCount}}": "{{updatedOperatorsCount}}/ {{totalOperatorsCount}}",
+  "Update status": "更新ステータス",
   "Subscription": "Subscription",
```

Comment:
If "Subscription" needs to be translated for both Japanese and Chinese?

Reply:
Looks like they updated this one.